base/src/ImageViewerModule.cpp

```diff
@@ -8,13 +8,73 @@
 #include "Logger.h"
 #include "Utils.h"
 
-class ImageViewerModule::Detail
+#if defined(__arm__) || defined(__aarch64__)
+#include "ApraNvEglRenderer.h"
+#include "DMAFDWrapper.h"
+#include "Command.h"
+#endif
+
+class DetailRenderer
 {
 
 public:
-	Detail(std::string& strTitle): mStrTitle(strTitle) {}
+	DetailRenderer(ImageViewerModuleProps& _props) : props(_props) {}
+
+	~DetailRenderer() 
+	{
+		#if defined(__arm__) || defined(__aarch64__)
+		 if(renderer)
```

Comment:
fix formatting

Reply:
call destroyWIndow and remove duplicate code

Reply:
removed it